wasm/main.go

```diff
@@ -1,69 +1,674 @@
-package main
+// GOOS=js GOARCH=wasm go build -o  ../assets/hive.wasm
 
-//go:generate GOOS=js GOARCH=wasm go build -o  ../assets/hive.wasm
+package main
 
 import (
+	"bufio"
 	"bytes"
 	"encoding/json"
+	"fmt"
+	"github.com/hako/durafmt"
+	logger "github.com/ipfs/go-log/v2"
 	"io/ioutil"
 	"net/http"
+	"reflect"
+	"strings"
 	"syscall/js"
-
-	logger "github.com/ipfs/go-log/v2"
+	"time"
 )
 
-var log = logger.Logger("sock/server")
+var log = logger.Logger("events")
+
 const (
+	EVENTS  = "http://localhost:4343/v3/events"
 	GATEWAY = "http://localhost:4343/v3/execute"
 )
 
-func ID() js.Func {
+var StartTime int64
+
+func Events() js.Func {
+
+	jsonfunc := js.FuncOf(func(this js.Value, args []js.Value) interface{} {
```

Comment:
There is no point of this variable declaration. For all your functions you can just have the format:

func FUNCNAME() js.Func {
    return js.FuncOf()
}

Save a few lines as well in the process. Problem is when I see jsonFunc declared here, I might go down to see if something else is being done on it before returning. When I see return in the first line itself, its better for understanding.